apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -928,6 +929,15 @@ export class RushConfiguration {
     return tempNamesByProject;
   }
 
+  /**
+   * Returns the pnpmfile filename for the version of PNPM supplied.
+   */
+  private static _getPnpmfileFilename(packageManagerToolVersion: string): string {
```

Comment:
Seems like this should be in a utility somewhere instead of in `RushConfiguration`

Reply:
The idea of keeping it here is that in general, you should be obtaining the path to the file from `RushConfiguration`, similar to how we get the shrinkwrap path, since you will also need the PNPM version which already comes from `RushConfiguration`.